x/stakeibc/keeper/reward_allocation_test.go

```diff
@@ -88,13 +104,26 @@ func (s *KeeperTestSuite) TestLiquidStakeRewardCollectorBalance_Success() {
 	s.FundModuleAccount(stakeibctypes.RewardCollectorName, sdk.NewCoin(IbcAtom, rewardAmount))
 	s.FundModuleAccount(stakeibctypes.RewardCollectorName, sdk.NewCoin(IbcOsmo, rewardAmount))
 
-	// Distribute rewards using new logic: 85% liquid staked to ConsumerToSendToProvider, 85% to auction
+	// Distribute rewards using new logic: 15% liquid staked to PoA validators, 85% to auction
 	s.App.StakeibcKeeper.AuctionOffRewardCollectorBalance(s.Ctx)
 
-	// Check ConsumerToSendToProvider module balance (should have liquid staked stTokens - 15% of original)
-	providerPortion := sdkmath.NewInt(150) // 15% of 1000
-	s.checkModuleAccountBalance(ccvtypes.ConsumerToSendToProviderName, StAtom, providerPortion)
-	s.checkModuleAccountBalance(ccvtypes.ConsumerToSendToProviderName, StOsmo, providerPortion)
+	// Check PoA validators received stTokens (15% of original amount gets liquid staked)
+	// Since the redemption rate is 1:1, 15% of 1000 = 150 stTokens total
```

Comment:
Do we need tests for edge cases: small amounts that don't divide evenly, zero balances, and validator payment failures. What happens with 1 token distributed to 7 validators?

Reply:
Good call, I'll add a 0 test case! (will cover zero balance and case where it truncates to 0)

The main test case has truncation already for the not dividing evenly case

It's not really possible to force the payment failure in tests

Reply:
actually the 0 case is already in there: `TestLiquidStakeRewardCollectorBalance_NoRewardsAccrued`